.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Can we get rid of this .vscode directory?

Reply:
If you remove this directory, you will not have the file to start the project in debug mode.

How are people going to debug the project?
Will each person who is going to use it have to create it over and over again?

Reply:
Totally code owner's preference, from `chore: Add vscode debug launch` I assumed it was debug code going into main. 
I did not see this committed previously and was of view it got committed by mistake.